weaver/wps_restapi/swagger_definitions.py

```diff
@@ -803,38 +875,69 @@ class LiteralDataDomainList(ExtendedSequenceSchema):
 
 class LiteralInputType(NotKeywordSchema, ExtendedMappingSchema):
     _not = [
-        WithFormats,
+        DescribeWithFormats,
     ]
     literalDataDomains = LiteralDataDomainList(missing=drop)
 
 
-class InputTypeDefinition(OneOfKeywordSchema):
+class DescribeInputTypeDefinition(OneOfKeywordSchema):
     _one_of = [
         # NOTE:
         #   LiteralInputType could be used to represent a complex input if the 'format' is missing in
         #   process deployment definition but is instead provided in CWL definition.
         #   This use case is still valid because 'format' can be inferred from the combining Process/CWL contents.
         BoundingBoxInputType,
-        ComplexInputType,  # should be 2nd to last because very permissive, but requires format at least
+        DescribeComplexInputType,  # should be 2nd to last because very permissive, but requires format at least
         LiteralInputType,  # must be last because it"s the most permissive (all can default if omitted)
     ]
 
 
-class InputType(AllOfKeywordSchema):
+class DeployInputTypeDefinition(OneOfKeywordSchema):
+    _one_of = [
+        # NOTE:
+        #   LiteralInputType could be used to represent a complex input if the 'format' is missing in
+        #   process deployment definition but is instead provided in CWL definition.
+        #   This use case is still valid because 'format' can be inferred from the combining Process/CWL contents.
+        BoundingBoxInputType,
+        DeployComplexInputType,  # should be 2nd to last because very permissive, but requires format at least
```

Comment:
Can you please replace the `'format'` by `'formats'` here while you are patching stuff? 
Its an old typo. 

See idem in above class.